pkg/virt-handler/vm.go

```diff
@@ -1272,9 +1272,17 @@ func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.Vi
 		networks[network.Name] = network.DeepCopy()
 	}
 	for _, iface := range vmi.Spec.Domain.Devices.Interfaces {
-		if iface.Bridge != nil && networks[iface.Name].Pod != nil {
-			return fmt.Errorf("cannot migrate VMI with a bridge interface connected to a pod network")
+		if networks[iface.Name].Pod != nil {
+			if iface.Masquerade != nil {
+				continue
+			}
+		}
+		if networks[iface.Name].Multus != nil {
+			if iface.Bridge != nil {
```

Comment:
We don't care about binding in secondary network. All should be migratable. You can `continue`, no matter what the binding is.

Reply:
Corrected